indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/utils/EnvironmentUtils.kt

```diff
@@ -17,24 +12,31 @@ internal object EnvironmentUtils {
             return testPoolIp ?: "127.0.0.1"
         }
 
+    //Should be similar to  RUST`s implementation
+    private val userHomePath: String get() = System.getenv("HOME")
```

Comment:
Could we make "HOME" more readable and less conflicted with other potential system environments?

Reply:
I don`t get your point